frontend/src/components/ReferralCreationModal/ReferralCreationModal.tsx

```diff
@@ -45,17 +57,39 @@ export default function ReferralCreationModal(props: ReferralCreationModalProps
     setActiveStep(activeStep + 1)
   }
 
-  const handleSubmit = () => {
+  const handleSubmit = async () => {
     // TODO: Send post request to backend
-    closeModal()
+    axios({
+      method: 'post',
+      url: '/referral',
+      headers: {
+        'Authorization': localStorage.getItem('authorization')
+      },
+      data: {
+        resumeFileName: resumeFilePath,
```

Comment:
Don't we need to post the actual resume somehow?

Reply:
@CosineP 
Yes, but I think we're backseating this for now because the resume stuff isn't required